### DISCUSSION TITLE
Download in emmett 2.6

### DISCUSSION THREAD

**Discussion by josejachuf** [emmett-framework/emmett#519] [Q&A] [answered]
Hi, I have de following code:

```python
@app.route("/download/<str:filename>")
async def download(filename):
    response.wrap_dbfile(db, filename)
```

> http://127.0.0.1:8000/download/fotografias.foto.bd94f39244258f23.312e706e67.png

I get **html** page with:

> Template /my-app/src/templates/download.html not found

I have to change the content type?

**Comment by gi0baro** [comment, marked as answer]
Wrap methods need a `return` statement, see the upgrade guide https://emmett.sh/docs/2.6.x/upgrading#version-26
